docs/index.md

```diff
@@ -0,0 +1,35 @@
+# TorchServe
+
+TorchServe is a flexible and easy to use tool for serving PyTorch models.
+
+> **WARNING:** TorchServe is experimental and subject to change.
+
+
+## Basic Features
+
+* [Serving Quick Start](TorchServe.html#serve-a-model) - Basic server usage tutorial
+* [Model Archive Quick Start](https://github.com/pytorch/serve/tree/master/model-archiver#creating-a-model-archive) - Tutorial that shows you how to package a model archive file.
+* [Installation](TorchServe.html#install-torchserve) - Installation procedures
```

Comment:
```suggestion
* [Installation](TorchServe.md#install-torchserve) - Installation procedures
```

Reply:
I removed TorchServe.html so it will directly link to https://github.com/pytorch/serve/blob/master/README.md#install-torchserve so it only needs to be maintained in one place